cfme/tests/intelligence/reports/test_reports_schedules.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+import pytest
+import yaml
+
+from cfme import test_requirements
+from cfme.utils.path import data_path
+
+schedules_report_dir = data_path.join("schedules_report")
+
+
+def files_schedules():
+    result = []
+    if not schedules_report_dir.exists:
+        schedules_report_dir.mkdir()
```

Comment:
**REQUIRED**
Ok, you've created a directory, obviously, it will be empty. So either remove this check or skip the test if the directory doesn't exist.